.eslintrc.json

```diff
@@ -24,7 +24,9 @@
 		"wp": true,
 		"wpApiSettings": true,
 		"window": true,
-		"document": true
+		"document": true,
+		"cy": true,
```

Comment:
We can put those two globals in their own `.eslintrc.json` file inside `cypress` folder.